translator/translate/otel/pipeline/appsignals/translator.go

```diff
@@ -54,6 +55,12 @@ func (t *translator) Translate(conf *confmap.Conf) (*common.ComponentTranslators
 	translators.Processors.Set(resourcedetection.NewTranslator(resourcedetection.WithDataType(t.dataType)))
 	translators.Processors.Set(awsappsignals.NewTranslator(awsappsignals.WithDataType(t.dataType)))
 
+	if conf.IsSet(common.DebugLogging) {
+		if enabled, _ := common.GetBool(conf, common.DebugLogging); enabled {
+			translators.Exporters.Set(logging.NewTranslator())
+		}
+	}
```

Comment:
nit: Don't need to check if it's set.
```suggestion
		if enabled, ok := common.GetBool(conf, common.DebugLogging); ok && enabled {
			translators.Exporters.Set(logging.NewTranslator())
		}
```